src/ray/protobuf/node_manager.proto

```diff
@@ -513,4 +520,8 @@ service NodeManagerService {
   // Failure: Is currently only used when grpc channel is unavailable for retryable core
   // worker clients. The unavailable callback will eventually be retried so if this fails.
   rpc IsLocalWorkerDead(IsLocalWorkerDeadRequest) returns (IsLocalWorkerDeadReply);
+  // Get the worker managed by local raylet.
+  // Failure: Sends to local raylet, so should never fail.
```

Comment:
we should still add error handling & retries just in case (there could be a logical bug in the raylet)